src/styles/_catalog.scss

```diff
@@ -1,22 +1,39 @@
+$cart-width: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
 
+  display: grid;
+  grid-auto-flow: row;
+  gap: 48px;
+  grid-template-columns: $cart-width;
   max-width: 944px;
   margin: 0 auto;
 
```

Comment:
We can make it a bit nicer for resolutions < 1024 with:
`justify-content: center;`